docs/data-ingestion/Proxmox.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Proxmox
+weight: 1
+sort: 1
+menu:
+  docs:
+    identifier: "proxmox"
+    parent: "data-ingestion"
+    weight: 1
+    # sort: 1
+aliases:
+  - /data-ingestion/proxmox.html
+  - /data-ingestion/Proxmox.html
+---
+
+# Proxmox Data Ingestion
+Since Proxmox Virtual Environment(PVE) and Proxmox Backup Server(PBS) support sending data using the InfluxDB We can use the InfluxDB write support built into VictoriaMetrics
+Currently PVE and PBS only support using an Authorization Token for authentication and does not support basic auth or a username and password.
+
+## Proxmox Virtual Environment (PVE)
+If want help Sending your data to Managed VictoriaMetrics check out [our blog](https://victoriametrics.com/blog/proxmox-monitoring-with-dbaas/).
+
+1. Login to PVE as an administrator
+2. Go to DataCenter > MetricServer > Add > InfluxDB
+3. Set the parameters as follows
+  * Name: VictoriaMetrics (can be changed to any string)
+  * Server: the hostname or IP of your VictoriaMetrics Instance
```

Comment:
does it require specifying the path? which VM port should user specify?

Reply:
you do not need to specify the influxdb write path but I haven't tested what happens if you need to modify the path for a multitenant setup.

the port is specified in a different form field in PVE but the url with the ip port is 1 form field in PBS I will add screenshots so this is more clear